packages/pwa/app/routes.jsx

```diff
@@ -81,30 +78,37 @@ const routes = [
         exact: true
     },
     {
-        path: '/:locale/cart',
+        path: '/cart',
         component: Cart,
         exact: true
     },
     {
-        path: '/:locale/product/:productId',
+        path: '/product/:productId',
         component: ProductDetail
     },
     {
-        path: '/:locale/search',
+        path: '/search',
         component: ProductList
     },
     {
-        path: '/:locale/category/:categoryId',
+        path: '/category/:categoryId',
         component: ProductList
     },
     {
-        path: '/:locale/account/wishlist',
+        path: '/account/wishlist',
         component: Wishlist
     },
+    {
+        path: '/:locale',
```

Comment:
We need to keep this route because it is the home page for other locales except home page.

Reply:
I vaguely remember this issue, but my brain is failing me a little. Can you explain the scenario and why this needs to be like that in a little more detail?

Reply:
Yes, this is for the scenario when the locale is set as `path`, and the user changes the locale (from the footer) to be another one (let's say it-IT), the homepage URL will now be /it-IT. We need this route here so it will regconize the homepage for that path. 

This is not necessary to be the case if the locale is set as `query_param`.  

I suppose we can add this route to the logic from the util function. 🤔